src/Maestrano/Configuration/WebhookConnec.cs

```diff
@@ -6,34 +6,28 @@ namespace Maestrano.Configuration
 {
     public class WebhookConnec
     {
-        public WebhookConnecSubscriptions Subscriptions { get; set; }
-
         /// <summary>
         /// Load Webhook into a WebhookAccount configuration object from a JObject 
         /// </summary>
         /// <returns>A WebhookAccount configuration object</returns>
         public static WebhookConnec LoadFromJson(JObject obj)
         {
             var config = new WebhookConnec();
-            config.NotificationsPath = obj["notification_path"].Value<string>();
-            var subscriptions = obj["connec_subscriptions"];
-            if (subscriptions != null)
-            {
-                config.Subscriptions = WebhookConnecSubscriptions.LoadFromJson(subscriptions.Value<JObject>());
-            }
-            else
-            {
-                config.Subscriptions = new WebhookConnecSubscriptions();
-            }
-
-            
+            config.NotificationPath = obj["notification_path"].Value<string>();
+            config.InitializationPath = obj["initialization_path"].Value<string>();
```

Comment:
We may want to double check whether the initialization_path this is still used or not. This may only be applicable to specific hosted apps for which we need to record an initialization endpoint to trigger the initial "full" sync.